index.html

```diff
@@ -62,6 +62,142 @@ <h2 class="second-header">I'm a Software Developer</h2>
             <div class="main-image"></div>
         </section>
     </main>
+
+    <section class="work-section">
+        <h2 class="second-title">My recent Works</h2>
+        <hr class="line_below_title">
+        <div class="card-container">
+            <div class="main_card">
+                <img class="image_from_main_card" src="./assets/images/cardImages/mainCard.png" alt="Work 1">
+                <h2 class="main-card-title">Multi-Post Stories</h2>
+                <p class="main-card-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the
+                    industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+                    text.
+                </p>
+                <ul>
+                    <li>CSS</li>
+                    <li>HTML</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button class="button-from-main-card" type="button">See project</button>
+            </div>
+        </div>
```

Comment:
- [ ] **[OPTIONAL]:** You did an excellent job applying the changes requested by the previous reviewer! 🥇 However, If you want to improve even more the semantics of your page, please consider using the `<article>` tag as a container for each card instead of `<div>`. The `<article>` is very useful when you have self-contained compositions, a card is a great example of this! 🤓 🔧 

```suggestion
        <article class="card-container">
            <div class="main_card">
                <img class="image_from_main_card" src="./assets/images/cardImages/mainCard.png" alt="Work 1">
                <h2 class="main-card-title">Multi-Post Stories</h2>
                <p class="main-card-description">A daily selection of privately personalized reads; no accounts or
                    sign-ups required. has been the
                    industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
                    text.
                </p>
                <ul>
                    <li>CSS</li>
                    <li>HTML</li>
                    <li>Bootstrap</li>
                    <li>Ruby</li>
                </ul>
                <button class="button-from-main-card" type="button">See project</button>
            </div>
        </article>
```

Reply:
@anagudelogu Wow! I didn't know that. Thank you! 
I appreciate your help a lot.